src/screens/FeedbackScreen.js

```diff
@@ -47,7 +50,8 @@ export const FormScreen = () => {
         email,
         phone,
         message,
-        consent
+        consent,
+        appInfo: appInfo
```

Comment:
this could be just `appInfo`, right?

Reply:
Yes, I renamed it and it seems like I missed that.